linkerd/meshtls/rustls/src/creds/store.rs

```diff
@@ -121,8 +126,9 @@ impl Store {
             NO_OCSP,
             now,
         )?;
-        debug!("Certified");
-        Ok(())
+
+        // verify the id as the cert verifier does not do that (on purpose)
```

Comment:
nit:
```suggestion
        // verify the identity, as the cert verifier does not do that (on purpose)
```
it might also be good if this comment explained _why_ we don't do that --- right now this just says it's "on purpose", but it doesn't say why...